extensions/adobe/experience/adcloud/search/adgroup-all.schema.json

```diff
@@ -0,0 +1,70 @@
+{
+    "meta:license": [
+        "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://ns.adobe.com/experience/adcloud/search/adgroup-all",
+    "title": "Adobe AdCloud Search Adgroup Full Extension",
+    "type": "object",
+    "meta:extensible": true,
+    "meta:abstract": true,
+    "meta:intendedToExtend": [
+        "https://ns.adobe.com/xdm/context/profile-adcloud"
+    ],
+    "description": "Adobe AdCloud Search Adgroup Full Extension. Contains all Solution added fields.",
+    "definitions": {
+        "adgroup-all":{
```

Comment:
Again, all these fields need to be in a namespace

Reply:
Added fields under proper namespace